PortalFetch/crawler_download.py

```diff
@@ -244,8 +280,11 @@ def main():
         fillAdvanceSearch(driver)
         # Save searched courses
         html = saveResult(driver)
-        filename = parser.get('config', 'filename')
-        quarter = parser.get('config', 'quarter')
+        # get quarter and fileName based On quarter_value in crawler.config
+        resultList = generateFilenameAndQuarter(value)
```

Comment:
1. The `generateFilenameAndQuarter` function should be rename to `generateQuarterAndFilename` since quarter is your first return value, and filename is the second return value. This will make your code more readable.
2. I would avoid having
```
resultList = generateFilenameAndQuarter(value)
filename = resultList[1]
quarter = resultList[0]
```
It is better to do 
```
quarter, filename = GenerateQuarterAndFilename(value)
```